client/chat/channel.tsx

```diff
@@ -515,6 +516,7 @@ export default function Channel(props: ChatChannelProps) {
               offline={sortedOfflineUsers}
             />
           }
+          modifyMenuItems={(...args) => addChannelMenuItems(...args, channelName)}
```

Comment:
nit: I think you actually could `useCallback` this safely now (since the memoization will be around creating the function reference, not the actual call to the function created)